test/test-core.js

```diff
@@ -1067,14 +1067,17 @@ describe('Test log-core', function () {
             overrideCustomFieldFormat("default");
         })
 
-
-
         it('Test timings', function () {
-            log("info","test");
-            var time1 = logObject.written_ts;
-            log("info","test");
-            var time2 = logObject.written_ts;
-            assert.isTrue(time1 < time2);
+            var timing_runs = 10000
```

Comment:
Is this enough to experience at least one overflow? I have no clue how fast the loop executes. But potentially it could finish in less than a millisecond.

Reply:
rewrote the tests to be deterministic to test failures due to random chance and "bad" timings